tools/deployments/validate-pr-description.ts

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable turbo/no-undeclared-env-vars */
+if (require.main === module) {
+	const errors = validateDescription(
+		process.env.TITLE as string,
+		process.env.BODY as string,
+		process.env.LABELS as string
+	);
+	if (errors.length > 0) {
+		console.error("Validation errors in PR description:");
+		for (const error of errors) {
+			console.error("- ", error);
+		}
+		process.exit(1);
+	}
+}
+
+export function validateDescription(
+	title: string,
+	body: string,
+	labels: string
+) {
+	const errors: string[] = [];
+
+	console.log("PR:", title);
+	const parsedLabels = JSON.parse(labels);
+
+	if (parsedLabels.includes("skip-pr-validation")) {
+		console.log(
+			"Skipping validation because the `skip-pr-validation` label has been applied"
+		);
+		return [];
+	}
+
+	if (
+		!/^Fixes (#\d+|\[[A-Z]+-\d+\]\(https:\/\/jira\.cfdata\.org\/browse\/[A-Z]+-\d+\))/m.test(
```

Comment:
This enforces links to Jira issues rather than just references, which should make it easier to cross reference